karabo_data/geometry2.py

```diff
@@ -70,6 +70,7 @@ def to_crystfel_geom(self, p, a):
             p=p,
             min_ss=(a * self.ss_pixels),
             max_ss=(((a + 1) * self.ss_pixels) - 1),
+            max_fs=self.fs_pixels,
```

Comment:
Unfortunately this isn't quite sufficient. This was written for AGIPD, where each tile spans the full fast-scan dimension (i.e. `min_fs == 0 and max_fs == (fs_pixels - 1)`). But in DSSC for instance, each tile spans the full slow-scan dimension, but only half of the fast-scan dimension.

As part of #142, I added a `_tile_slice` method on classes which would define what section of the data (in fast-scan & slow-scan dimensions) a given tile corresponds to. We'd probably want to use that or something similar to determine the min/max fs/ss for CrystFEL.

Reply:
Yes I've added a method to get the dimensions of a tile

Reply:
I am not 100% sure about the dimensions for LPD though. To be honest I do not 100% understand the code to get the dimensions for the distortion. Maybe you might want to have a look and tell me if that's ok. I could not check with hdfsee because of the issue @ebadkamil posted. But since this is solved now maybe @ebadkamil could try creating a VDS again ;) ?

Reply:
There is now a new virtual dataset that has all the 16 modules. Same location in scratch folder.

Reply:
Yes, that seems to work! Thx @ebadkamil

Reply:
All three detectors seem to work.